version/version.go

```diff
@@ -0,0 +1,6 @@
+// Package version handles k6 versioning
+package version
+
+// SemVer contains the current version of k6
+// represented using Semantic Versioning expression.
+const SemVer = "0.56.0"
```

Comment:
I'm not super convinced `SemVer` is really the best choice here, is there any reason why not to keep `Version`?
Do we really expect this package to contain multiple "formats" of the k6 version?

I'd like to see what other colleagues think.

Reply:
>Do we really expect this package to contain multiple "formats" of the k6 version?

Not really, but it still makes sense for me because it's what it contains. A version expressed in semantic versioning format.

Reply:
Well, I guess one could argue that `version.SemVer` may also express that there are other representations of the k6 version, like `version.Full` (referred to as `vX.Y.Z` for instance), which isn't the case. 

Anyway, I don't see major problems going with this one, and we can always add a `Version` if needed, preferred. So, I'm going to give it a 👍🏻

Reply:
I share the same feeling as @joanlopez, it was also one of the reasons why I raised the thread https://github.com/grafana/k6/pull/4537#discussion_r1954796281

Reply:
@joanlopez @olegbespalov Just to clarify, now that we clarified the inconvenience of `k6.Version` what is the request here? To have `version.Version`?

Reply:
Yeah, that would have probably been my first choice if anybody asked me.

Reply:
`Version` also works for me :+1:

Reply:
Does `k6` have to be on the root directory? Although not ideal, for the complications we confronted, why not create another `k6` package and put `Version` in it? `version.Version` stutters. I believe `version.SemVer` is more descriptive than that. I don't have a strong opinion, though. Just my two cents.

Reply:
>Does a k6 package have to be on the root directory?

@inancgumus Not, but what's a good package to not overcomplicate it too much? `go.k6.io/k6/k6.Version`? I don't have any rationale to be against it but I'm definitely not excited about it?
What's your proposal?

Reply:
_Technically_ it could be `go.k6.io/k6/cmd.Version`, not sure thought about imports issues :shrug:

Reply:
>Technically it could be go.k6.io/k6/cmd.Version,

I like the idea! @olegbespalov :tada: It works for me, and it makes sense about the current program structure. @inancgumus @joanlopez How does it sound for you?

Reply:
That sounds like a good idea.

Reply:
if this will not be needed to be public one as we move otel output inside k6 , can we just ove it to internal/lib/consts ?

Reply:
>what about meta.Version (go.k6.io/k6/meta.Version)?

I'm fine with that, but I would find difficult to explain what it should contain. What is the difference with the current `lib` package? There is the risk to add another junk drawer similar to `lib` or `common`, where I would find it difficult to understand if the code I might want to add in the future fits it or not.

For example, to stay concrete, can you provide a description that you would put as documentation for the `meta` package?

By the way, I have three additional suggestions:

1. `semver.Version`
2. `buildinfo.Version`
3. `appversion.Version`

Do you prefer one of them?

Reply:
@mstoykov Yes, we can do it, and it's a good idea if we merge OTel. But one of my original goals were also to use this pull request to find a better place for those constants instead to stay in a generic `lib` package.

So, I think it still makes sense to have `go.k6.io./k6/internal/buildinfo.Version`.

Reply:
I believe `meta` is more descriptive than a utility-like package `lib`. Another one could be `metadata`. But, still, they can turn out to be grab-all. So, I also suggested using a `build` package, which "provides" k6-build-related information, like version.

```go
// Package build provides information about the k6 binary at build time.
package build

// Version is the current semantic version of k6.
const Version = "0.57.0"
```

`semver` and `appversion` are too narrow. I'm also fine with `buildinfo`, but I'm a fan of more concise names.

Reply:
Ok, I've incorporated all the suggestions and I think we've reached a good balance.